modules/environment-roles/templates/tdr_jenkins_read_params.json.tpl

```diff
@@ -0,0 +1,22 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Action": "ssm:DescribeParameters",
+      "Resource": "*"
+    },
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Action": "ssm:GetParameters",
+      "Resource": [
+        "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/admin/*",
```

Comment:
It this for backwards-compatibility, and it will be deleted once Jenkins uses its own service client?

Reply:
It needs the secrets values so they can be set in the configuration JSON at container startup and on updating. The configuration JSON does not contain the secret values when it is exported.

Reply:
I see, thanks.

As discussed elsewhere, it's a shame that you can't reference environment variables when you import realm configuration via the API!